app/profile/createTask/page.tsx

```diff
@@ -58,11 +58,7 @@ export default function CreateTask() {
       importanceIsSelected
     ) {
       try {
-        const response = await axios.post(
-          // `https://notqe.vercel.app/api/usersTasks`,
-          "http://localhost:3000/api/usersTasks",
-          { userID: userData.userId, task: task }
-        );
+        await tasksApi.post("", { userID: userData.userId, task: task });

```

Comment:
Właśnie dlatego powinnismy unikać dodawania w clientApi zbyt wielu części urla, bo potem tutaj mamy sytuacje ze jest pusty string ktory nie mowi nic :(

Reply:
poza tym spróbowałbym tutaj sprowadzić to do notacji `{ userId, task }` czyli musisz to userId wyjąć z userData i zmienić nazwe paramsa w body (userID zmienic na userId lub odwrotnie)
Tak aby po prostu przekazane body było takie: `{ userId, task }`  czytelne